llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -816,6 +822,19 @@ def CreateHandle : DXILOp<57, createHandle> {
   let attributes = [Attributes<DXIL1_0, [ReadOnly]>];
 }
 
+def CBufferLoadLegacy : DXILOp<59, cbufferLoadLegacy> {
+  let Doc = "reads from a TypedBuffer";
```

Comment:
```suggestion
  let Doc = "reads from a constant buffer";
```

Reply:
Nice catch. I updated this to use the doc wording from [DXIL.rst](https://github.com/microsoft/DirectXShaderCompiler/blob/main/docs/DXIL.rst#operations-via-external-functions)